.github/workflows/cd.yml

```diff
@@ -0,0 +1,74 @@
+name: CD
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Kibana
+      uses: actions/checkout@v2
+      with:
+        path: kibana
+        repository: elastic/kibana
+        fetch-depth: 0
+    - name: Use release
+      run: |
+        cd ./kibana
+        git checkout tags/v7.9.1 -b v7.9.1
+    - name: Get node and yarn versions
+      id: versions
+      run: |
+        echo "::set-output name=node_version::$(cat ./kibana/.node-version)"
+        echo "::set-output name=yarn_version::$(cat ./kibana/package.json | jq '.engines.yarn' | tr -d '"')"
```

Comment:
`jq -r '.engines.yarn' ./kibana/package.json` should be equivalent to  `cat ./kibana/package.json | jq '.engines.yarn' | tr -d '"'` in this case